client/src/components/Format/Format.tool.ts

```diff
@@ -0,0 +1,42 @@
+import { FormatValue } from '.'
+
+export type Options = {
+  value: number
+  intlOptions: { maximumFractionDigits: number }
+  postfix: string
+}[]
+
+export const DEFAULT_WEIGHT_OPTIONS = {
+  value: 0,
+  intlOptions: { maximumFractionDigits: 2 },
+  postfix: ' g',
+}
+
+export const DEFAULT_SIZE_OPTIONS = {
+  value: 0,
+  intlOptions: { maximumFractionDigits: 2 },
+  postfix: ' bytes',
+}
+
+export const WEIGHT_OPTIONS = [
+  { value: 1, intlOptions: { maximumFractionDigits: 2 }, postfix: ' g'},
+  { value: 1_000, intlOptions: { maximumFractionDigits: 2 }, postfix: ' kg'},
+  { value: 1_000_000, intlOptions: { maximumFractionDigits: 2 }, postfix: ' t'},
+]
+
+export const SIZE_OPTIONS = [
+  { value: 1, intlOptions: { maximumFractionDigits: 2 }, postfix: ' bytes'},
+  { value: 1_024, intlOptions: { maximumFractionDigits: 2 }, postfix: ' kilobytes'},
+  { value: 1_024*2, intlOptions: { maximumFractionDigits: 2 }, postfix: ' megabytes'},
+  { value: 1_024*3, intlOptions: { maximumFractionDigits: 2 }, postfix: ' gigabytes'},
+  { value: 1_024*4, intlOptions: { maximumFractionDigits: 2 }, postfix: ' terabytes'},
+]
+
+export const getMaxOptions = (
+  value: FormatValue,
+  options: Options,
+  defaultOption?) => {
+  return options.findLast((option: { value: number }) => value / option.value >= 1) ?? defaultOption
+}
+
+export const capitalizeFirstLetter = (v: string) => String(v).trim().charAt(0).toUpperCase() + String(v).trim().slice(1)
```

Comment:
can we create also `export const toString = (v: string) => String(v).trim()` and replace in formatters too